README.md

```diff
@@ -0,0 +1 @@
+# portfolio-micro
```

Comment:
**[REQUIRED CHANGES]**

- Please, let's update our readme to make it more professional, as of right now it missing some important headers. refer to the[microverse readme template](https://github.com/microverseinc/readme-template) to add the missing header to make it more professional

Reply:
Thanks, I just delete it back. Because I do not think we need a README file in a branch.